signer/threshold_validator.go

```diff
@@ -263,11 +263,12 @@ func (pv *ThresholdValidator) waitForPeerSetEphemeralSharesAndSign(
 	peerStartTime := time.Now()
 	defer wg.Done()
 	peerEphemeralSecretParts := make([]CosignerEphemeralSecretPart, 0, pv.threshold-1)
-	for _, EncryptedSecrets := range *encryptedEphemeralSharesThresholdMap {
-		for _, ephemeralSecretPart := range EncryptedSecrets {
+
+	for _, encryptedSecrets := range encryptedEphemeralSharesThresholdMap {
+		for _, ephemeralSecretPart := range encryptedSecrets {
 			// if share is intended for peer, check to make sure source peer is included in threshold
 			if ephemeralSecretPart.DestinationID == peer.GetID() {
-				for thresholdPeer := range *encryptedEphemeralSharesThresholdMap {
+				for thresholdPeer := range encryptedEphemeralSharesThresholdMap {
 					if thresholdPeer.GetID() == ephemeralSecretPart.SourceID {
 						// source peer is included in threshold signature, include in sharing
 						peerEphemeralSecretParts = append(peerEphemeralSecretParts, ephemeralSecretPart)
```

Comment:
No change because this is existing code that is already working. But there are 3 for loops (2 nested) and 2 nested if statements. I bet there's a refactor that could make this more readable in the future.